lib/build-tokens.js

```diff
@@ -56,9 +56,11 @@ async function buildTokensCommand(commandArgs) {
     source: tokensSource
       ? [`${tokensSource}/core/**/*.json`, `${tokensSource}/core/**/*.toml`]
       : [],
+    preprocessors: ['tokens-studio'],
```

Comment:
[inform] See `sd-transforms` README for more information on the [`tokens-studio` preprocessor](https://github.com/tokens-studio/sd-transforms?tab=readme-ov-file#using-the-preprocessor).